azure_pipelines/automation.yml

```diff
@@ -145,3 +145,51 @@ jobs:
         issue_body = '''@AzureAD/appleidentity \nAutomation failed for [$(repositoryName)]({0}) ran against commit : {1} \n Pipeline URL : [{2}]({2})'''.format('$(Build.Repository.Uri)', git_commit, pipeline_uri) 
         github.create_issue(github_org, repo, issue_title, issue_body, labels=['automation failure'])
   
+
+
+- job: cocoapods_lib_lint
+  displayName: Run Cocoapods lib lint
+  timeoutInMinutes: 30
+  cancelTimeoutInMinutes: 5
+  strategy:
+    maxParallel: 2
+    matrix:
+      STATIC_LIBRARY: 
+        args: "--fail-fast --swift-version=5 --allow-warnings --use-static-frameworks"
+      DYNAMIC_LIBRARY: 
+        args: "--fail-fast --swift-version=5 --allow-warnings"
+  workspace:
+    clean: all
+  
+  steps:
+
+  - checkout: self
```

Comment:
Are you checking out the repository in the pipeline again? Don't we already do checkout at line 40?

Reply:
If I remove the checkout part (or specify `checkout: none`, the job fails. It seems [it's not in the directory anymore](https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=1298344&view=logs&j=6134a362-24cf-5c9f-d760-bb2ddc5a4da1&t=905a9e23-2c92-5939-3fe4-8a495bb67b00).

I think it's because these two jobs are executed in parallel.